build/phing/build.xml

```
@@ -273,8 +283,8 @@
                 standard="${repo.root}/build/vendor/drupal/coder/coder_sniffer/Drupal/"
                 showSniffs="true"
                 showWarnings="true"
-                haltonerror="true"
```

This is not a fix, this would just be ignoring the errors no?